exporters/trace/gcp_exporter/internal/recordable.cc

```diff
@@ -1,12 +1,47 @@
 #include "exporters/trace/gcp_exporter/recordable.h"
-
+#include "opentelemetry/context/threadlocal_context.h"
 
 OPENTELEMETRY_BEGIN_NAMESPACE
 namespace exporter
 {
 namespace gcp
 {
 
+constexpr size_t kAttributeStringLen = 256;
+constexpr size_t kAnnotationStringLen = 256;
```

Comment:
kAnnotationStringLen seems unused.

Reply:
This would be used once we add Events to the GCP recordable.
Please see [this](https://github.com/census-instrumentation/opencensus-cpp/blob/master/opencensus/exporters/trace/stackdriver/internal/stackdriver_exporter.cc#L128) for the use-case.

Reply:
True, but it is not used in current code, right? 
I think it would be better not to add it here to make changes modular and self sufficient - so each PR only introduces variables that are used with it.

Reply:
Alright, I'll add it in another PR then.
Thank you!